app/controllers/concerns/vet360/writeable.rb

```diff
@@ -27,7 +27,7 @@ def write_to_vet360_and_render_transaction!(type, params, http_verb: 'post')
       record = build_record(type, params)
       validate!(record)
       response = write_valid_record!(http_verb, type, record)
-      create_user_audit_log(type) if PROFILE_AUDIT_LOG_TYPES.keys.include?(type)
+      create_user_audit_log(type) if PROFILE_AUDIT_LOG_TYPES.keys.include?(type.to_sym)
```

Comment:
Could we just check if we get a value instead of checking if the key exists? 
```ruby
create_user_audit_log(type) if PROFILE_AUDIT_LOG_TYPES[type].present?
```